cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -223,10 +225,13 @@ function (google_cloud_cpp_enable_cleanup)
 
     list(REMOVE_DUPLICATES GOOGLE_CLOUD_CPP_ENABLE)
 
-    set(GOOGLE_CLOUD_CPP_ENABLE_GRPC ON)
-    if ("${GOOGLE_CLOUD_CPP_ENABLE}" STREQUAL "storage")
-        set(GOOGLE_CLOUD_CPP_ENABLE_GRPC OFF)
-    endif ()
+    set(GOOGLE_CLOUD_CPP_ENABLE_GRPC OFF)
+    foreach (feature ${GOOGLE_CLOUD_CPP_ENABLE})
```

Comment:
Consider: `foreach (feature IN LISTS GOOGLE_CLOUD_CPP_ENABLE)`

Reply:
Fixed the source I copied from.